src/lib/services/feature-toggle-service-v2.ts

```diff
@@ -96,6 +96,17 @@ class FeatureToggleServiceV2 {
         this.featureEnvironmentStore = featureEnvironmentStore;
     }
 
+    validateProject(
```

Comment:
I will change this to actually take the projectId and featureName as input and validate them against the store.